logging/adapters/tendermint_log15/convert.go

```diff
@@ -0,0 +1,70 @@
+package adapters
+
+import (
+	"time"
+
+	"github.com/eris-ltd/eris-db/logging/loggers"
+	"github.com/eris-ltd/eris-db/logging/structure"
+	. "github.com/eris-ltd/eris-db/util/slice"
```

Comment:
In general I would agree. In this case these functions are really intended for brevity and providing builtin-style functions for working with generic slices. I introduced them because there is a lot of that with the generic `Log(keyvals... interface{}) error` kitlog interface. I could be persuaded to step back, but I quite like it how it is.